README.md

```diff
@@ -6,3 +6,25 @@ Terraform module to provision an [`RDS Aurora`](https://aws.amazon.com/rds/auror
 ## License
 
 Apache 2 License. See [`LICENSE`](LICENSE) for full details.
+
+
+## Usage
+
+Include this repository as a module in your existing terraform code:
+
+```terraform
+module "rds_cluster_aurora_acig" {
+  source              = "git::https://github.com/cloudposse/terraform-aws-rds-cluster.git?ref=tags/0.2.1"
```

Comment:
should use the `master` branch in `README` examples.
git::https://github.com/cloudposse/terraform-aws-rds-cluster.git?ref=master